pkg/dns-server/resolver/handler.go

```diff
@@ -0,0 +1,56 @@
+package resolver
+
+import (
+	"fmt"
+
+	"github.com/miekg/dns"
+)
+
+func (d *SimpleDNSResolver) parseQuery(m *dns.Msg) {
+	for _, q := range m.Question {
+		switch q.Qtype {
+		case dns.TypeA:
+			simpleDNSLog.Info("Query for %s\n", q.Name)
+
+			ip, err := d.ForwardLookup(q.Name)
+			if err != nil {
+				simpleDNSLog.Error(err, "Unable to resolve ", "q.Name", q.Name)
```

Comment:
nitpicking: `simpleDNSLog.Error(err, "Unable to resolve", "name", q.Name)` (space after resolve, and just "name")